src/Plugin/GraphQL/DataProducer/Entity/EntityLoad.php

```diff
@@ -36,6 +37,18 @@
  *       label = @Translation("Entity bundle(s)"),
  *       multiple = TRUE,
  *       required = FALSE
+ *     ),
+ *     "access" = @ContextDefinition("boolean",
+ *       label = @Translation("Check access"),
+ *       required = FALSE
+ *     ),
+ *     "access_user" = @ContextDefinition("entity:user",
+ *       label = @Translation("User"),
+ *       required = FALSE
+ *     ),
+ *     "operation" = @ContextDefinition("string",
```

Comment:
Can you call this access_operation?

Reply:
Sure thing. I copied the names fomr the `entity_access` producer, but this makes it clearer.

Reply:
The entity_access producer should also use the object notation now that I looked at it again. :-P

Reply:
:) I can open a new PR if you would prefer not to muddy this one, or if you don't have a real preference I can make those similar changes in this PR.